### ISSUE TITLE
Unit tests fail when using Build.isDev

### ISSUE THREAD

**Issue by simonhaenisch** [ionic-team/stencil#1213]
**Stencil version:**

```
 @stencil/core@0.15.2
```

**I'm submitting a:**

[x] bug report
[ ] feature request
[ ] support request

**Current behavior:**

Running the tests fails when using `Build.isDev` anywhere in a component:

> TypeError: Cannot read property 'isDev' of undefined

**Expected behavior:**

I guess there are two possible solutions: 

* The build conditionals should also be replaced when running tests (but not sure whether the compiler runs before spec tests?).
* Provide a mock for `Build` (sth like `{ Build: { isDev: false } }`).

**Related code:**

```tsx
import { Build } from '@stencil/core';

if (Build.isDev) {
  // ...
}
```

**Other information:**

This is the code that replaces it for normal builds:

https://github.com/ionic-team/stencil/blob/c78fe511976923e84f36b70e6e4fbfe7939f7924/src/compiler/bundle/rollup-bundle.ts#L21-L28

**Comment by eshtadc** [comment]
I have a work-around in place that my code utilizes my own constant that I set based on the `Build.isDev` in case it helps anyone who finds this.

```
let isDev;
try {
  isDev = Build.isDev;
} catch {
  isDev = false;
}
export const isDevBuild:boolean = isDev;
```

**Comment by eshtadc** [comment]
This appears to be resolved in 1.0.0. Can someone confirm that this is true?

**Comment by simonhaenisch** [comment]
Yup that’s right, thanks for reminding me about that issue (: Before that it was also possible to provide a mock for `Build`, but it’s not needed anymore.
